crates/nostr/src/event/mod.rs

```diff
@@ -90,6 +93,14 @@ impl Event {
     pub fn as_json(&self) -> String {
         serde_json::json!(self).to_string()
     }
+
+    /// Initialize the ots optional field in this event with the OpenTimestamps proof,
+    /// according to NIP-03.
```

Comment:
Better wording: "Timestamp this event with OpenTimestamps, according to NIP-03"

Reply:
changed in latest commit